### PR TITLE
[14_0_X] Change unsubjet_map in btag info producers to tracked

### PR DESCRIPTION
**Pull request by stahlleiton**
#### PR description:

backport of https://github.com/cms-sw/cmssw/pull/44591

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

@mandrenguyen

### CODE REVIEW

**Comment by mandrenguyen** [comment]
type btv

**Comment by mandrenguyen** [comment]
backport of https://github.com/cms-sw/cmssw/pull/44591

**Comment by mmusich** [comment]
see please https://github.com/cms-sw/cmssw/pull/44591#issuecomment-2033718480

File: HLTrigger/Configuration/python/customizeHLTforCMSSW.py [hunk 1]
```diff
@@ -253,6 +253,15 @@ def customizeHLTfor44511(process):
             delattr(producer, 'AXOL1TLModelVersion')
     return process
 
+def customizeHLTfor44591(process):
```
Comment (mmusich) [inline HLTrigger/Configuration/python/customizeHLTforCMSSW.py:256]:
```suggestion
def customizeHLTfor44597(process):
```

File: HLTrigger/Configuration/python/customizeHLTforCMSSW.py [hunk 2]
```diff
@@ -253,6 +253,15 @@ def customizeHLTfor44511(process):
...
+    """
+    Customisation for running HLT with the updated btag info producers from the PR 44591
```
Comment (mmusich) [inline HLTrigger/Configuration/python/customizeHLTforCMSSW.py:258]:
```suggestion
    Customisation for running HLT with the updated btag info producers from the PR 44597
```

File: HLTrigger/Configuration/python/customizeHLTforCMSSW.py [hunk 3]
```diff
@@ -273,4 +282,6 @@ def customizeHLTforCMSSW(process, menuType="GRun"):
     # customizes AXOL1TL condition in the L1 menu
     process = customizeHLTfor44511(process)
 
+    process = customizeHLTfor44591(process)
```
Comment (mmusich) [inline HLTrigger/Configuration/python/customizeHLTforCMSSW.py:285]:
```suggestion
    process = customizeHLTfor44597(process)
```

**Review by mmusich — COMMENTED** [review]
it's not super-important, but for book-keeping.

**Comment by mandrenguyen** [comment]
+1

**Pull request merged**